llvm/utils/TableGen/RegisterBankEmitter.cpp

```diff
@@ -7,7 +7,7 @@
 //===----------------------------------------------------------------------===//
 //
 // This tablegen backend is responsible for emitting a description of a target
-// register bank for a code generator.
+// register bank and register bank info for a code generator.
```

Comment:
This is very vague, you should elaborate a bit on what info it's generating.

Reply:
I added a basic test which I'll expand on later. As for the comment, I see your point but I didn't expand on it there. Instead added a pointer to the RFC.